modules/openapi-generator/src/main/resources/csharp-netcore/libraries/httpclient/ApiClient.mustache

```diff
@@ -214,6 +214,7 @@ namespace {{packageName}}.Client
             _httpClient = new HttpClient(_httpClientHandler, true);
             _disposeClient = true;
             _baseUrl = basePath;
+            #warning
```

Comment:
probably to remove, this generates a c# warning directive without a message

Reply:
Ah yes, that was an accident I'll remove it